### PR TITLE
Add availability payload

### PR DESCRIPTION
**Pull request by timstanley1985**
**Description:**
Documentation for MQTT switch availability payload

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>. TBC

### CODE REVIEW

File: source/_components/switch.mqtt.markdown [hunk 1]
```diff
@@ -35,9 +35,11 @@ Configuration variables:
 - **name** (*Optional*): The name of the switch. Default is 'MQTT Switch'.
 - **state_topic** (*Optional*): The MQTT topic subscribed to receive state updates.
 - **command_topic** (*Required*): The MQTT topic to publish commands to change the switch state.
-- **availability_topic** (*Optional*): The MQTT topic subscribed to receive availability (online/offline) updates. The payload is the same for the command_topic.
+- **availability_topic** (*Optional*): The MQTT topic subscribed to receive availability (online/offline) updates.
 - **payload_on** (*Optional*): The payload that represents enabled state. Default is "ON".
 - **payload_off** (*Optional*): The payload that represents disabled state. Default is "OFF".
+- **payload_available** (*Optional*): The payload that represents available state. Default is "ON".
+- **payload_not_available** (*Optional*): The payload that represents unavailable state. Default is "OFF".
```
Comment (frenck) [inline source/_components/switch.mqtt.markdown]: Missing words, maybe replace it with:

> The payload that represents an unavailable state. The default is "OFF".

File: source/_components/switch.mqtt.markdown [hunk 2]
```diff
@@ -35,9 +35,11 @@ Configuration variables:
 - **name** (*Optional*): The name of the switch. Default is 'MQTT Switch'.
 - **state_topic** (*Optional*): The MQTT topic subscribed to receive state updates.
 - **command_topic** (*Required*): The MQTT topic to publish commands to change the switch state.
-- **availability_topic** (*Optional*): The MQTT topic subscribed to receive availability (online/offline) updates. The payload is the same for the command_topic.
+- **availability_topic** (*Optional*): The MQTT topic subscribed to receive availability (online/offline) updates.
 - **payload_on** (*Optional*): The payload that represents enabled state. Default is "ON".
 - **payload_off** (*Optional*): The payload that represents disabled state. Default is "OFF".
+- **payload_available** (*Optional*): The payload that represents available state. Default is "ON".
```
Comment (frenck) [inline source/_components/switch.mqtt.markdown]: Missing words, maybe replace it with:

> The payload that represents an available state. The default is "ON".

Comment (Landrash) [inline source/_components/switch.mqtt.markdown]: There's only one available state @frenck

File: source/_components/switch.mqtt.markdown (hunk 1 above)
Comment (Landrash) [inline source/_components/switch.mqtt.markdown]: There's only one available state @frenck

File: source/_components/switch.mqtt.markdown (hunk 2 above)
Comment (frenck) [inline source/_components/switch.mqtt.markdown]: I'm sorry, you are right, in that case, I would suggest:

> The payload that represents the available state. The default is "ON".

**Pull request merged**